index.bs

```diff
@@ -3544,7 +3545,13 @@ laid out as shown in <a href="#table-authData">Table <span class="table-ref-foll
                 - Bit 2: [=User Verified=] ([=UV=]) result.
                     - `1` means the user is [=user verified|verified=].
                     - `0` means the user is not [=user verified|verified=].
-                - Bits 3-5: Reserved for future use (`RFU2`).
+                - Bit 3:
+                    - `1` means the [=Public Key Credential Source|credential source=] is allowed to be backed up 
```

Comment:
This formulation leaves an "impossible" state: xxx10xxx would mean that it _is_ backed up, but it's not allowed to be. I liked Akshay's four states:

00 &rarr; will never be backed up
01 &rarr; could be backed up, but backup is not currently happening.
10 &rarr; will soon be backed up.
11 &rarr; is backed up.

This text merges the 01 and 10 states, but I think it's worth having the difference since we're consuming limited flag space.

Reply:
We can go either way on this.

Couple of reasons for current PR proposal
- Till now all the bits had individual meanings. And @ve7jtb proposal in #1692 seemed cleaner. PR just refines the wording of it. 
    - Single Device 0 / multi-device capable 1
    - Not backed up 0 / backed up 1
- "will soon be backed up" and "could be backed up, but backup is not currently configured." have same action items for a relying party. So "will soon be backed up" seemed unnecessary.